.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: Test
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        meteorRelease:
+          - '--release 1.4.4.5'
+          - '--release 1.5.4.1'
+          - '--release 1.6.0.1'
+          - '--release 1.7.0.5'
+          - '--release 1.8.1'
+          - '--release 1.8.3'
+          - '--release 1.9.1'
+          - '--release 1.10.2'
+          - '--release 1.11'
+          - '--release 1.12.1'
+          - '--release 2.1.1'
+          - '--release 2.2'
+          - '--release 2.3.2'
+          # Latest version
+          - 
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
```

Comment:
Should also include Node 14.x for Meteor 2.3+ releases.

Reply:
Given the previous comment of yours, I may just add checks only for the 2.3.x releases along with 14.x for the sake of brevity.

Reply:
Regarding the node version, it won't matter since [mtest](https://github.com/zodern/mtest) invokes the tests using the meteor binary so it uses the appropriate version regardless.